### ISSUE TITLE
Object.activate is deprecated.

### ISSUE THREAD

**Issue by gepoch** [gepoch/linter-write-good#10]
AtomLinter v0.X.Y API has been deprecated. Please refer to the Linter docs to update and the latest API: https://github.com/AtomLinter/Linter/wiki/Migrating-to-the-new-API
```
Object.activate (/home/gmarchin/.atom/packages/linter/lib/main.coffee:57:8)
Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:240:19)
<unknown> (/usr/share/atom/resources/app.asar/src/package.js:221:30)
Package.measure (/usr/share/atom/resources/app.asar/src/package.js:165:15)
Package.activate (/usr/share/atom/resources/app.asar/src/package.js:213:14)
PackageManager.activatePackage (/usr/share/atom/resources/app.asar/src/package-manager.js:452:21)
```

**Comment by Stephen304** [comment]
:+1:

**Comment by gepoch** [comment]
@Stephen304 Hey! I'm working on a fix for this at the moment. I might be able to get to it tonight, even!

**Comment by Stephen304** [comment]
Sweet - The 4 deprecation warnings from all the linter packages I have are beginning to be an eyesore :sweat_smile:

**Comment by gepoch** [comment]
Yeah. It's no good.

Sadly, the linter API is pretty much entirely different, so it's pretty much a rewrite. Fortunately, this is a super tiny plugin.

**Comment by gepoch** [comment]
@Stephen304 fixed in 0.7.0!

Let me know if you run into it again somehow..
